spec/serializers/placement_serializer/v01_spec.rb

```diff
@@ -16,7 +13,8 @@
         created_at
         updated_at
       ].each do |field|
-        expect(json.keys).to include(field)
+        expect(json.keys).to include(field), "Expected key '#{field}' to be present, but it was not"
+        expect(json[field]).to be_present, "Expected field '#{field}' to have a value, but it was '#{json[field]}'"
```

Comment:
Ummm, 

`Expected field '#{field}' to have a value, but it was '#{json[field]}'`

What kinda blank value was causing pain?

```suggestion
        expect(json[field]).to be_present, "Expected field '#{field}' to have a value, but it was '#{json[field]}'"
```